### PR TITLE
feature: add object property editor

### PR DESCRIPTION
**Pull request by bzp2010**
add feature #168 
> Make object properties available for graphical editing
> 图形化编辑对象属性

### CODE REVIEW

File: src/components/PluginDialog/index.vue [hunk 1]
```diff
@@ -282,6 +316,16 @@ export default class extends Vue {
           break
         case 'object':
           schemaKeys[key] = {}
+          this.objectPropertiesArray[key] = []
+          if (this.pluginData[key]) {
+            let objectKeys = Object.keys(this.pluginData[key])
+            objectKeys.map((item) => {
```
Comment (juzhiyuan) [inline src/components/PluginDialog/index.vue:322]: Please remove the extra `()`

Comment (juzhiyuan) [reply src/components/PluginDialog/index.vue:322]: And use `Object.keys().map()` directly.

File: src/components/PluginDialog/index.vue [hunk 2]
```diff
@@ -355,6 +401,56 @@ export default class extends Vue {
...
+    return data
+  }
+
+  /**
+   * Force rerender on object property content changed
+   * @param key
+   * @param value
+   * @param isKey
+   */
+  private onObjectPropertyChange(key: any, value: any, isKey: boolean) {
```
Comment (juzhiyuan) [inline src/components/PluginDialog/index.vue:449]: those parameters are unused?

Comment (bzp2010) [reply src/components/PluginDialog/index.vue]: oh. some careless😄

File: src/components/PluginDialog/index.vue (hunk 1 above)
Comment (bzp2010) [reply src/components/PluginDialog/index.vue]: fixed

**Pull request merged**